sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClient.java

```diff
@@ -807,6 +809,10 @@ private HttpPipeline setupPipeline(HttpClient httpClient) {
             builder.claims(customClaimRequest);
         }
 
+        if (loginHint != null) {
+            builder.loginHint(loginHint);
```

Comment:
MSAL in .NET has a "None" option for the `Prompt` option so that the account selection is not even presented. Is there an equivalent in Java?

Reply:
Synced offline - there's no `NO_PROMPT` option in msal4j thus for now we will always ask the user to select. Issue opened to support this once msal4j adds support: https://github.com/Azure/azure-sdk-for-java/issues/22727